WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -975,6 +975,11 @@ extension AztecPostViewController {
             })
             return
         }
+
+        if neeedsVerificationBeforePosting() {
```

Comment:
I really don't like the fact that the logic for this has to be repeated at least three times: here, in the `WPPostViewController` and in `WPLegacyPostViewController` (and potentially more that I don't know about), but I honestly have no clue how to streamline this. ideas very much welcome!

Reply:
Yeah... It has me wishing that Swift Protocol Extensions played nice with ObjC.   That said, we're about ready to sunset the legacy and hybrid editors so if we just focused on the Aztec editor it would be fine.  Probably no need to implement the change on code that is destined for the 🗑.

Reply:
+1 for fixing only for Aztec